app/models/pglogical_subscription.rb

```diff
@@ -199,15 +192,20 @@ def self.find_id(to_find)
   end
   private_class_method :find_id
 
+  def self.pg_connection
```

Comment:
This is provides a specific method for fetching the `connection.raw_connection`, since this is the one way the the two `.pglogical` methods differed between the two classes.

I probably could have shared this method with the implementation that is in `MiqPglogical`, since `connection.raw_connection` is probably the same as `ApplicationRecord.connection.raw_connection`, but in the off chance it isn't, I decided to make this explicit in each class.